windows/src/engine/kmcomapi/processes/keyboard/Keyman.System.MitigateWin10_1803LanguageInstall.pas

```diff
@@ -0,0 +1,89 @@
+// See discussion online at https://github.com/keymanapp/keyman/issues/1285
+// Introduced in Keyman 11. Back ported to Keyman 10. If Microsoft introduces
+// a fix in a later release, we will need to update this to turn the mitigation
+// off for that version. In that situation, users will need to reinstall their
+// language profile in order to get the corrected language.
+unit Keyman.System.MitigateWin10_1803LanguageInstall;
+
+interface
+
+uses
+  Winapi.Windows;
+
+type
+  TMitigateWin10_1803 = class
+  type
+    TLanguageReference = record
+      Code: LANGID;
+      BCP47: string;
+      Name: string;
+    end;
+
+    TMitigatedLanguage = record
+      OriginalLanguage: TLanguageReference;
+      NewLanguage: TLanguageReference;
+    end;
+
+    class function IsMitigationRequired(Code: LANGID; var lang: TMitigateWin10_1803.TMitigatedLanguage): Boolean;
+  end;
+
+implementation
+
+
+const
+  MitigatedLanguages: array[0..2] of TMitigateWin10_1803.TMitigatedLanguage = (
+    (OriginalLanguage: (Code: $005E; Name: 'Amharic'); NewLanguage: (BCP47: 'gez-Ethi-ET'; Name: 'Geez')),
+    (OriginalLanguage: (Code: $0073; Name: 'Tigrinya'); NewLanguage: (BCP47: 'gez-Ethi-ET'; Name: 'Geez')),
+    (OriginalLanguage: (Code: $005B; Name: 'Sinhala'); NewLanguage: (Code: $0409; Name: 'English (US)'))
```

Comment:
Is `$0409` the WIndows code for English (US)?

Just wondering if it's clearer to use `BCP47: 'en'`

Reply:
I deliberately used $0409 because I didn't want to do a conversion from 'en'; when you install the language it needs the integer value anyway...